code/game/antagonist/station/renegade.dm

```diff
@@ -15,19 +15,32 @@ var/datum/antagonist/renegade/renegades
 	initial_spawn_target = 6
 
 	var/list/spawn_guns = list(
-		/obj/item/weapon/gun/energy/gun,
 		/obj/item/weapon/gun/energy/laser,
-		/obj/item/weapon/gun/projectile,
-		/obj/item/weapon/gun/projectile/revolver/detective,
+		/obj/item/weapon/gun/energy/captain,
+		/obj/item/weapon/gun/energy/lasercannon,
+		/obj/item/weapon/gun/energy/xray,
+		/obj/item/weapon/gun/energy/gun,
+		/obj/item/weapon/gun/energy/gun/burst,
+		/obj/item/weapon/gun/energy/gun/nuclear,
+		/obj/item/weapon/gun/energy/crossbow,
+		/obj/item/weapon/gun/energy/crossbow/largecrossbow,
+		/obj/item/weapon/gun/projectile/automatic,
+		/obj/item/weapon/gun/projectile/automatic/mini_uzi,
 		/obj/item/weapon/gun/projectile/automatic/c20r,
-		/obj/item/weapon/gun/projectile/deagle/camo,
+		/obj/item/weapon/gun/projectile/automatic/sts35,
+		/obj/item/weapon/gun/projectile/automatic/wt550,
+		/obj/item/weapon/gun/projectile/automatic/z8,
+		/obj/item/weapon/gun/projectile/automatic/as24,
```

Comment:
I'm fairly sure giving renegades (or anyone for that matter) an automatic shotgun with 24 slugs loaded and **burst fire** is a bad idea.